### DISCUSSION TITLE
refund

### DISCUSSION THREAD

**Discussion by REXSUNDER** [aclap-dev/video-downloadhelper#1065] [License & Purchase requests]
### What's your issue?

I recently purchased a VideoDownloader Helper license on Sun Jul 28, 2024, with Google Chrome. Unfortunately, I am experiencing issues with downloading videos due to a conflict with Google services that the Chrome version does not allow me to download YouTube It appears that switching to Firefox might resolve this issue how do I get a refund.

### The 10 first digits (only 10) of your license number or email address

332a6fdcb3

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.
